interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
+ //Get new_validate option to check if global new validation is on.
+ $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+ //Gets validation rules from Page Validation list.
+ $collectthis = collectValidationPageRules("/interface/forms/newpatient/new.php");
+ ?>
+ <?php if($new_validate && $collectthis): ?>
```

Comment:
We want to give an option for the user to use the old validation. The 'collectValidationPageRule' method doesn't check if new validation is on or off (in globals). If I were to just check $collectthis, then even if new validation option (in globals) is off, the new validation would occur (if the validation itself is active in 'Page Validation' list.